### ISSUE TITLE
Runtime Error: 3 invalid profile(s) for extrude feature

### ISSUE THREAD

**Issue by polygonfuture** [DVE2000/Dogbone#23]
Hi all,

I'm having an issue with using dogbones on a project.   I am trying to use it at the component stage.  I've already laid out  my material but realized, I need to add dogbone after the fact.

It is not working at all.

Here is a screen cap of my error:

![dogboe_problem](https://cloud.githubusercontent.com/assets/12026545/24226408/60cede04-0f23-11e7-9b2d-2a98ec05eb8b.PNG)

![problem_piece](https://cloud.githubusercontent.com/assets/12026545/24226441/879e2922-0f23-11e7-9c24-7afab0f843ca.PNG)

Any ideas?

**Comment by jinschoi** [comment]
I had this same issue when I'm trying to use components to lay out parts for a chair. The Up Plane does not lie parallel to any of the world planes. I tried aligning the body within the component to the component's origin, but that still gives the error when trying to apply the dogbone at the end of the history; it looks like the dogbone add in only respects the world origin. I was able to resolve this by rolling the timeline back to before the component was put into the assembly, but after I had aligned it, and running the dogbone there.
